client/data/marketplace/use-wpcom-plugins-query.ts

```diff
@@ -41,9 +41,12 @@ export const getWPCOMPluginsQueryParams = (
 	type: Type,
 	searchTerm?: string,
 	tag?: string
-): [ QueryKey, QueryFunction< { results: Array< { [ plugin: string ]: Plugin } > } > ] => {
+): [ QueryKey, QueryFunction< any[] > ] => {
```

Comment:
Any way we could avoid `any` here? (maybe `unknown` if we really don't know the type?)

Reply:
Fair point. 
I replaced it with `unknown`.

Reply:
Actually, using `unknown` start breaking type-checks in other files (such as this [one here](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/marketplace/pages/marketplace-plugin-install/index.tsx#L300)).

I restored the `any` type but we'll want to follow up and define the normalized plugin type ([here](https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/plugins/utils.js#L186)).